lib/src/native/realm_core.dart

```diff
@@ -2179,6 +2245,13 @@ class _RealmCore {
       return completer.future;
     });
   }
+
+  void immediatelyRunFileActions(App app, String realmPath) {
+    using((arena) {
+      _realmLib.invokeGetBool(() => _realmLib.realm_sync_immediately_run_file_actions(app.handle._pointer, realmPath.toCharPtr(arena)),
+          "An error occurred while deleting Realm file. Check if the file is in use: '$realmPath'");
```

Comment:
```suggestion
          "An error occurred while deleting the Realm file. Check if the file is in use: '$realmPath'");
```
should we replace `deleting the` with `working with the`

Reply:
the `immediatelyRunFileActions` does not imply deletions but that's probably what is going on. still we could be vague about it.

Reply:
"An error occurred while resetting the Realm. Check if the file is in use: